arviz/data/base.py

```diff
@@ -2,10 +2,9 @@
 from copy import deepcopy
 import datetime
 import warnings
-
-import numpy as np
 import pkg_resources
 import xarray as xr
+import arviz.utils as utils
```

Comment:
These should still be relative imports. I believe `from . import utils` should work (or `..` )

Reply:
The reason why btw is that in some scenarios `import arviz` could import the sys installed version of arviz rather than the modules that are relative to this package. By specifying relative import were sure of what code is being imported